pulser-core/pulser/sequence/_seq_drawer.py

```diff
@@ -698,7 +698,7 @@ def phase_str(phi: Any) -> str:
                         alpha=0.3,
                         hatch="////",
                     )
-                else:
+                else:  # pragma: no cover
```

Comment:
Why is this needed ?

Reply:
Basically this case is only touched when using `draw_sequence()` with modulation and a sampling rate. While `Simulation()` was doing this, `QutipEmulator()` uses `draw_samples()`. I could delete it but it might come in handy again in the future, so I just kept it

Reply:
Sure, let's keep it then ;)